sympy/physics/control/lti.py

```diff
@@ -1300,6 +1300,50 @@ def to_expr(self):
             return Pow(self.den, -1, evaluate=False)
 
 
+class PIDController(TransferFunction):
+    """
+    A PID Controller class that subclasses TransferFunction.
+
+    Parameters
+    ==========
+    KP : Expr, Number
+        Proportional gain.
+    KI : Expr, Number
+        Integral gain.
+    KD : Expr, Number
+        Derivative gain.
+    s : Symbol
+        The complex frequency variable.
+
+    Examples
+    ========
+    >>> from sympy.abc import s
+    >>> from sympy.physics.control.lti import PIDController
+    >>> KP, KI, KD = 1, 0.1, 0.01
+    >>> pid = PIDController(KP, KI, KD, s)
+    >>> pid
```

Comment:
Kindly test it out well. Ensure that the `doctest` output is correct on your terminal first, then change the implementation.
```
File "/home/runner/work/sympy/sympy/sympy/physics/control/lti.py", line 1324, in sympy.physics.control.lti.PIDController
Failed example:
    pid
Expected:
    TransferFunction(K_D*s + K_P + K_I/s, 1, s)        # What you expect
Got:
    PIDController(1, 0.1, 0.01, s)                     # What sympy returns
```